packages/tailwind/tailwind.spec.ts

```diff
@@ -46,6 +49,38 @@ describe('tailwind component', () => {
         '"<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd"><div class="bg-test text-white" style="background-color: rgb(252,186,3); color: rgb(255,255,255);">Hello world</div><div class="bg-test" style="color: rgb(255,255,255); background-color: rgb(252,186,3);">custom background</div>"',
       )
     })
+
+    it('should render Tailwind styles correctly if text contains the word `container`', async () => {
+      const component = h(Tailwind, [
+          h(Html, [
+            h(Head),
+            h(Body,  { class: 'w-full max-w-[650px] md:p-7'}, [
+              h(
+                'div',
+                [
+                  'Text contains the word container that brooked Tailwind styles'
+                ]
+              ),
+              h(
+                'div',
+                {
+                  class: 'text-[#42B883]',
+                },
+                [
+                  'custom background'
+                ]
+              ),
+            ])
+        ])
+      ])
+
+
+      const actualOutput = await render(component)
+
+      expect(actualOutput).toMatchInlineSnapshot(
+        `"<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd"><html lang="en" dir="ltr"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta name="x-apple-disable-message-reformatting"></meta></meta></head><body class="md:p-7" style="width: 100%; max-width: 650px;"><div>Text contains the word container that brooked Tailwind styles</div><div class="text-[#42B883]" style="color: rgb(66,184,131);">custom background</div></body></html>"`,
```

Comment:
whoops, just noticed that the meta tags has a bug with the close tags like from issue #215   🌚